src/Components/Assets/AssetsList.tsx

```diff
@@ -341,18 +341,28 @@ const AssetsList = () => {
                       </span>
                     </p>
                   </div>
-                  <div className="md:flex justify-between pt-2">
-                    <div className="md:flex flex-wrap">
+                  <div className=" px-4 md:flex justify-between pt-2">
```

Comment:
```suggestion
                  <div className="px-4 md:flex justify-between pt-2">
```
Extra whitespace at the start of className